src/views/Profile/AdminTable/AdminTextField.tsx

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import FormControl from '@material-ui/core/FormControl';
 import TextField from '@material-ui/core/TextField';
 import { FieldEnum, IField } from '../../../models/adminLab/types';
+import { TextFieldsEnum } from '../../../models/experts/types';
 
 interface IMaterialsTextField {
-  field: FieldEnum;
+  field: FieldEnum | TextFieldsEnum;
```

Comment:
I don't see `TestFieldsEnum` in https://github.com/ita-social-projects/dokazovi-fe/blob/epic_14_Authors_list_and_personal_info/src/models/experts/types.ts

what do I miss?

Reply:
![Screenshot 2022-10-12 at 00 04 12](https://user-images.githubusercontent.com/40029364/195197949-0dbce3dc-7142-470b-9494-97211701a1e7.png)